.travis.yml

```diff
@@ -32,12 +39,18 @@ install:
     - pip install --no-index --find-links=travis_wheels -r requirements/compiled.txt
     - pip install --find-links=base_wheels -r requirements/prod.txt -r requirements/dev.txt
     - pip install coveralls
+    # JS test requirements
+    - curl -sL https://deb.nodesource.com/setup | sudo bash -
+    - sudo apt-get install -y nodejs
```

Comment:
That's good to know -- I'll remove the installation of nodejs from the `.travis.yaml` config and see if presents a problem